### PR TITLE
Add .nvmrc file, Prettier support for Nunjucks and use SCSS

### PR DESCRIPTION
**Pull request by davidmiddletonmoj**
Add .nvmrc file with node version set to `18`
Update npm engine version to `^9`
Add `prettier-plugin-jinja-template` as dev dependency plus config
Refactor `.sass` files to `.scss` for consistency

### CODE REVIEW

File: .nvmrc [hunk 1]
```diff
@@ -0,0 +1 @@
+18
```
Comment (andrewrlee) [inline .nvmrc:1]: meh.. missing new line

**Review by andrewrlee — DISMISSED** [review]
LGTM

File: .nvmrc (hunk 1 above)
Comment (davidmiddletonmoj) [reply .nvmrc]: Thanks. Corrected.

**Review by andrewrlee — APPROVED** [review]
LGTM

**Comment by ushkarev** [comment]
i got excited about prettier dealing with nunjucks files (i love code style consistency) but having tried it in our project, it proposed this change

prettier nunjucks

🤣🤣🤣

… so actually, i'm happy that this PR did not _enforce_ running prettier on nunjucks views! (i.e. via `lint-staged` config in package.json)

**Comment by davidmiddletonmoj** [comment]
Oh dear, that's very odd.
I've personally not seen any issues with it formatting HTML badly, but it hasn't been perfect with Nunjucks "blocks" (`{{ }}`, {% %}`, etc)
Sometimes it removes blank lines between blocks and it doesn't appear to tidy up bad nesting.
Hence why I left it as an optional, manual thing for now 😀

**Pull request merged**